Classes/Login/LoginSplashViewController.swift

```diff
@@ -144,8 +143,13 @@ final class LoginSplashViewController: UIViewController, GithubSessionListener {
         safariController?.dismiss(animated: trueUnlessReduceMotionEnabled)
         state = .fetchingToken
 
-        client.requestAccessToken(code: code) { result in
-            self.handle(result: result, authMethod: .oauth)
+        client.requestAccessToken(code: code) { [weak self] result in
+            switch result {
+            case .error:
```

Comment:
🤔, why is this `.error` and not `.failure`?

Reply:
Also, why do you handle the `.error` here first, when in other places you handle `.success` first?

Reply:
Two different `Result`s and the naming diverged